### ISSUE TITLE
Réduire l'overfitting

### ISSUE THREAD

**Issue by SimWalther** [SimWalther/vietnam_coffee_detection#14]
**Il y a plusieurs possibilités :**
1) c'est la structure du réseau de neurone qui provoque de l'overfitting.
-> il faudrait alors rajouter une couche de BatchNormalization, ajouter du dropout au niveau de la convolution...
2) c'est les features choisies
-> explorer quelles features donnent le moins d'overfitting
